RELEASE.md

```diff
@@ -0,0 +1,4 @@
+Release type: patch
```

Comment:
```suggestion
Release type: minor
```

I'd mark this as minor just to make it clearer it does update the schema (since the URLs have been updated)

@bellini666 what do you think?

Reply:
Makes sense. Even though this is mostly a non breaking change, it can (and will) break schema dumping tests